src/main/java/de/tum/in/www1/artemis/config/JacksonConfiguration.java

```diff
@@ -36,7 +36,9 @@ public Jdk8Module jdk8TimeModule() {
      */
     @Bean
     public Hibernate6Module hibernate6Module() {
-        return new Hibernate6Module();
+        Hibernate6Module module = new Hibernate6Module();
+        module.disable(Hibernate6Module.Feature.USE_TRANSIENT_ANNOTATION);
```

Comment:
please remove this change

Reply:
This change is necessary

Reply:
Why is that change necessary. If I understand it correctly, all `@Transient` annotations are now useless